src/lib/tasks/app.rake

```diff
@@ -131,19 +131,38 @@ namespace :app do
       presenter.restrict_after(time, weight)
     end
   end
-    
-  
-  desc 'create a schedule for most recent event'
+
+  desc 'clear the current schedule (DANGER: Irreversible!!!). You must do the before generating a new schedule'
+  task clear_schedule: :environment do
+    STDOUT.puts "Are you sure? This destroys the existing schedule and you will not be\n" \
+      "able to retrieve it. You should back up the database before doing this.\n\nIf you are really sure, type \"SCHEDULE ARMAGEDDON\" now (anything else to cancel)..."
+    input = STDIN.gets.strip
+    if input == 'SCHEDULE ARMAGEDDON'
+      event = Event.current_event
+      event.sessions.each { |s| s.update(timeslot_id: nil) }
```

Comment:
A little more straight forward might be to use `update_all`

```
Event.current_event.sessions.update_all("timeslot_id = NULL")
```
or whatever the appropriate syntax. The loop is fine but might take a second or two.